src/pages/docs/layout/blocks.mdx

```diff
@@ -288,3 +288,41 @@ In the grid demo example, each block will take up an entire row at the smallest
 </div>
 ```
 </Codemarkup>
+
+## [Spacing](#spacing)
+
+You can add spacing utilities to nested blocks by adding the classes to the block-container. You need to add `blocks {spacer-class}`. The `blocks` class is needed, without it you will simply be adding the spacers to the block-container rather than the nested blocks. This can be a great method for spacing multiple blocks equally. Adding `blocks {spacer-class}` will ensure that the spacing fits the container along the x-axis. Checkout the example below. The block-container width increases based on the x-axis padding that is added. 
```

Comment:
```suggestion
You can add spacing utilities to nested blocks by adding the classes to the block-container. You need to add `blocks {spacer-class}`. The `blocks` class is needed. Without it, you will simply be adding the spacers to the block-container rather than the nested blocks. This can be a great method for spacing multiple blocks equally. Adding `blocks {spacer-class}` will ensure that the spacing fits the container along the x-axis. Check out the example below. The block-container width increases based on the x-axis padding that is added. 
```